src/MoneyMarket.java

```diff
@@ -20,6 +20,12 @@ public MoneyMarket(Profile holder, double balance, boolean isLoyal) {
     }
 
 
+    public double getBalance(){
```

Comment:
this does the same thing in every subclass why write it 5 times when you can just write it in the super class?